dashboard/js/app/nls/bg/monitor.js

```diff
@@ -37,9 +37,7 @@ define({
     encodeLatencyMax: "Максимално закъснение на енкодера",
     transportLatency: "Закъснение на предаването",
     decodeLatency: "Закъснение на декодера",
-    fecPercentage: "Общ процент на корекция на грешки напред (FEC)",
-    fecFailureTotal: "Общо неуспех на FEC",
-    fecFailureInSecond: "FEC грешки/сек",
+    packetsLostTotal: "Общо неуспех на FEC",
```

Comment:
All leftover translations are wrong since they still reference FEC. I suggest you to remove them completely. The new dashboard will be english only.

Reply:
Actually, I've just noticed that there's already the `packetsLostTotal` key that was translated but not shown anywhere in the dashboard. I'll simply remove the one related to FEC.